.changeset/config.json

```diff
@@ -1,8 +1,13 @@
 {
   "$schema": "https://unpkg.com/@changesets/config@3.0.2/schema.json",
-  "changelog": ["@changesets/changelog-github", { "repo": "terrazzoapp/terrazzo" }],
+  "changelog": [
+    "@changesets/changelog-github",
+    {
+      "repo": "terrazzoapp/terrazzo"
+    }
+  ],
   "commit": false,
-  "linked": [["@terrazzo/cli", "@terrazzo/parser"]],
+  "fixed": [["@terrazzo/cli", "@terrazzo/parser", "@terrazzo/token-tools"]],
```

Comment:
I think this might be the source of my pain—I think I want `fixed`, not `linked` 🤔. But we’ll see.